src/SamplesApp/SamplesApp.Shared/App.xaml.cs

```diff
@@ -314,6 +314,7 @@ private void ActivateMainWindow()
 #if DEBUG && (__SKIA__ || __WASM__)
 			_mainWindow!.EnableHotReload();
 #endif
+			//await Task.Delay(3000); // Artificial delay to simulate asynchronous activation
```

Comment:
Are we removing this before merging in?

Reply:
This is mainly a note for future verifications of the launch lifecycle with `async`, so I think it is useful to keep it in